sql/src/test/java/io/crate/executor/transport/TransportShardDeleteActionTest.java

```diff
@@ -70,7 +71,7 @@ public void prepare() throws Exception {
 
         transportShardDeleteAction = new TransportShardDeleteAction(
             Settings.EMPTY,
-            MockTransportService.local(Settings.EMPTY, Version.V_5_0_1, THREAD_POOL),
+            MockTransportService.local(Settings.EMPTY, Version.V_5_0_1, THREAD_POOL, null),
```

Comment:
pls also change to use Version.CURRENT (really don't know why one fixed it to a concrete version :()